src/Admin/Polyfills.php

```diff
@@ -128,6 +128,10 @@ public function register_shimmed_scripts( $wp_scripts ) {
 	 * @param WP_Styles $wp_styles The WP_Styles instance for the current page.
 	 */
 	public function register_shimmed_styles( $wp_styles ) {
+		if ( version_compare( get_bloginfo( 'version' ), '5.4', '<' ) ) {
```

Comment:
Unrelated change: Noticed some undesirable CSS effects when working in 5.2 locally, so this will force the polyfilled wp-components styles. Note this only runs on AMP screens.